RecFCCeeCalorimeter/src/components/CellPositionsHCalPhiThetaSegTool.cpp

```diff
@@ -27,90 +27,29 @@ StatusCode CellPositionsHCalPhiThetaSegTool::initialize()
     error() << "Unable to locate Geometry service." << endmsg;
     return StatusCode::FAILURE;
   }
-  // get PhiTheta segmentation
```

Comment:
I wonder if we could have kept the positioning tool for the phi-theta fictional grid readout as is (maybe useful for other detector concepts?) and create a separate positioning tool for the new phi-"theta" readout of the ALLEGRO HCal? Maybe more a question for @BrieucF. If we don't care about the positioning tools for the "grid-like" segmentations then we could also remove the CellPositionsECalBarrelPhiThetaSegTool.h/cpp ...

Reply:
the positioning tool for phi-theta grid-like HCal segmentation, as it is without my changes, heavily depends on the HCal geo builder code. Therefore, I am not sure if it would be useful for other detector concepts.